timvt/db/tiles.py

```diff
@@ -47,21 +48,21 @@ async def _tile_from_bbox(self, bbox: BoundingBox, columns: str) -> bytes:
                 SELECT
                     ST_Segmentize(
                         ST_MakeEnvelope(
-                            $1,
-                            $2,
-                            $3,
-                            $4,
-                            $5
+                            :xmin,
+                            :ymin,
+                            :xmax,
+                            :ymax,
+                            :epsg
                         ),
-                        $6
+                        :segSize
                     ) AS geom
             ),
             mvtgeom AS (
                 SELECT ST_AsMVTGeom(
-                    ST_Transform(t.{geometry_column}, $5),
+                    ST_Transform(t.{geometry_column}, CAST(:epsg AS INT)),
```

Comment:
Should remove the f-string vars and instead bind them directly to the `sqlalchemy.sql.text` query